ops/terraform/services/base/values/prod.yaml

```diff
@@ -108,8 +108,12 @@
 /bfd/${env}/common/nonsensitive/rds_master_username: bfduser
 /bfd/${env}/common/nonsensitive/rds_cluster_identifier: bfd-prod-aurora-cluster
 /bfd/${env}/common/nonsensitive/rds_instance_class: db.r6i.12xlarge
-/bfd/${env}/common/nonsensitive/rds_instance_count: "3"
 /bfd/${env}/common/nonsensitive/rds_security_group: bfd-prod-aurora-cluster
+/bfd/${env}/common/nonsensitive/rds_min_reader_nodes: 1
+/bfd/${env}/common/nonsensitive/rds_max_reader_nodes: 3
+/bfd/${env}/common/nonsensitive/rds_scaling_cpu_target: 75
+/bfd/${env}/common/nonsensitive/rds_scale_in_cooldown: 300
+/bfd/${env}/common/nonsensitive/rds_scale_out_cooldown: 300
```

Comment:
Curious what others' opinions are on these values. These are the _default_ values for CPU-based application autoscaling, and they work _fine_ when testing against an ephemeral database cluster. But, I'm not sure if these values are _too_ conservative or not.

I certainly think we want our "scale in cooldown" to remain high just the databases don't scale-in unnecessarily, but perhaps we could do with a lower value for the "scale out cooldown"? As it stands, it takes around 15 minutes between each scale-out event, which is a _bit_ long.

Thoughts?

Reply:
I used a 50% threshold as part of the analysis earlier this year to approximate the potential savings in this strategy. I think you could reduce that threshold to 50% and we can go ahead an evaluate this in time.

Reply:
Agreed. I think we can be more aggressive in when we scale out because we're simply not scaling _at all_ just yet. I'll change it to 50% in all environments.

Reply:
Done in commit [0f7a9b3](https://github.com/CMSgov/beneficiary-fhir-data/pull/2393/commits/0f7a9b358f4817da07c8c94ba35024a5744a2c52).